resources/views/components/password-strength.blade.php

```diff
@@ -0,0 +1,70 @@
+@slots(['label'])
+
+@php
+    $length = Rapidez::config('customer/password/minimum_password_length', 8);
+    $checks = [
+        'length' => [
+            'text' => __('Bevat :length karakters', ['length' => $length]),
+            'regex' => '^.{' . $length . ',}',
+        ],
+        'lowercase' => [
+            'text' => __('Contains a lowercase letter'),
+            'regex' => '[a-z]',
+        ],
+        'uppercase' => [
+            'text' => __('Contains an uppercase letter'),
+            'regex' => '[A-Z]',
+        ],
+        'number' => [
+            'text' => __('Contains a number'),
+            'regex' => '[\d]',
+        ],
+        'special' => [
+            'text' => __('Contains a special character'),
+            'regex' => '[^a-zA-Z0-9]',
+        ],
+    ];
+@endphp
+
+<div>
+    <x-rapidez::label class="text-inactive mb-2">
+        {{ $label }}
+    </x-rapidez::label>
+    <password-strength
+        v-slot="passwordStrength"
+        v-bind:checks='@json($checks)'
+        v-bind:min-classes="{{ Rapidez::config('customer/password/required_character_classes_number', 3) }}"
```

Comment:
Both Magento configs should also go into https://github.com/rapidez/core/blob/master/config/rapidez/magento-defaults.php as per https://github.com/rapidez/core/pull/782